src/ast.rs

```diff
@@ -199,9 +222,9 @@ pub struct If {
 
 impl Compile for If {
     fn compile(&self, t: &Target) -> String {
-        unimplemented!()
+        return format!("If {} ( \n {} ) \n ELSE ( {} \n ) \n", 
```

Comment:
A few things here:
1. Try to use implicit returns where possible (its more "rustic")
2. Try to keep casing the same everywhere (make `If` all caps)
3. The placement of the `\n`s results in trailing spaces. Try to tidy up your whitespace